README.md

```diff
@@ -16,8 +16,14 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
-```
+It gets rid of child elements. You create a class of "clearfix" and then use the following syntax.
+.clearfix:after {
+  content: "."
+  visibility: hidden;
+  display: block;
+  height: 0;
+  clear: both;
+}
 
```

Comment:
Clear fix clears elements of its children if the parent container is not floated.